### ISSUE TITLE
Confusion about the Single GPU.

### ISSUE THREAD

**Issue by Odelllll** [ldkong1205/LaserMix#31]
I am training on 10% of the SemanticKITTI dataset using a single GPU for the Cylinder3D model. In order to maintain consistency with the experimental results presented in your paper, should I modify the batch size, learning rate, and iteration count? If modifications are necessary, could you please provide guidance on the appropriate adjustments?

**Comment by ldkong1205** [comment]
Hi @Odelllll, thanks for your interest in this work!

For semi-supervised learning with 10% labels: As long as:
1) the number of labels used is the same;
2) the segmentation backbone used is the same; 

There is no need to maintain consistency with training configurations (batch size, learning rate, iterations, etc.). 

You can find our preprocessed data-splitting files from [here](https://github.com/ldkong1205/LaserMix/blob/main/docs/DATA_PREPARE.md#semantickitti).

Feel free to let me know if you have further questions!

**Comment by Odelllll** [comment]
Thank you for your answer!
